platform/ui/src/components/ToolbarButton/ToolbarButton.tsx

```diff
@@ -39,6 +39,11 @@ const ToolbarButton = ({
 
   const activeClass = isActive ? 'active' : '';
   const shouldShowDropdown = !!isActive && !!dropdownContent;
+  const iconEl = icon ? (
+    <Icon name={icon} />
+  ) : (
+    <div>{label || 'Missing icon and label'}</div>
```

Comment:
Does this mean that a toolbar button can now be a simple label?

Reply:
Yes